inputs/test-fixtures/heatmap.yaml

```
@@ -0,0 +1,40 @@
+dashboards:
+  - name: "[Test] Heatmap - Geographic Traffic"
+    description: "Test fixture for heatmap visualization"
+    panels:
+      # ES|QL variant
+      - type: charts
+        title: "Traffic Heatmap by Geographic Location"
+        grid: { x: 0, y: 0, w: 24, h: 15 }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)